main/solution/machine-images/config/infra/files/rstudio/nginx.conf

```diff
@@ -65,20 +65,12 @@ http {
     proxy_set_header X-Real-IP $remote_addr;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
-   # Health check url
-    location /health {
-      return 200 'OK';
-      add_header Content-Type text/plain;
-    }
-
     location / {
       proxy_pass http://localhost:8787;
-      proxy_redirect https://localhost:8787/ $scheme://$http_host/;
     }
 
     location /auth-do-sign-in {
       proxy_pass http://localhost:8787;
-      proxy_redirect https://localhost:8787/ $scheme://$http_host/;

```

Comment:
Why was this removed?

Reply:
These are changes needed for RstudioV2. Since we are going to have RstudioV2 in RL repository, reverted this file back to original version.